index.html

```diff
@@ -0,0 +1,217 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Timothy | Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```

Comment:
Great job adding the hamburger👍. Kindly wrap your hamburger menu in a `nav` tag since the purpose of this section is to provide navigation links. It will be semantically correct if it is placed in a nav tag. 😎⚔️